src/components/Reports/ExpectedMonthlyTotalReport/Table/ExpectedMonthlyTotalReportTable.tsx

```diff
@@ -0,0 +1,133 @@
+import React, { useState } from 'react';
+import {
+  Box,
+  Typography,
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TableRow,
+  Paper,
+  Accordion,
+  AccordionSummary,
+  AccordionDetails,
+  Link,
+} from '@material-ui/core';
+import { ExpandMore } from '@material-ui/icons';
+import { useTranslation } from 'react-i18next';
+
+export interface Contact {
+  contact: string;
```

Comment:
```suggestion
  name: string;
```

I think `name` makes more sense than `contact` here.